walk-in-interview/src/main/webapp/new-job.js

```diff
@@ -114,6 +206,7 @@ submitButton. addEventListener('click', (_) => {
       .then((repsonse) => repsonse.text())
       .then((data) => {
         console.log('data', data);
+        document.getElementById(NEW_JOB_ERROR_MESSAGE.ID).innerText = '';
```

Comment:
when there is error below, should we also set this element text?

Reply:
I thought I already do this?